### ISSUE TITLE
Importing existing Astra Table fails with `Error: error getting table` with existing database, keyspace and table details.

### ISSUE THREAD

**Issue by jonathanjuursema** [datastax/terraform-provider-astra#366]
This is a follow-up for:
- https://github.com/datastax/terraform-provider-astra/issues/355

I'm performing the exact same actions as described in the referenced issue, but instead of a plug-in panic, I'm now encountering an error.

### Terraform Version
```bash
➜  git:(main) ✗ terraform -version
Terraform v1.7.4
on darwin_arm64
+ provider registry.terraform.io/datastax/astra v2.2.9
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/vault v3.12.0

```

### Affected Resource(s)
- `astra_table`

### Terraform Configuration Files
Identical to https://github.com/datastax/terraform-provider-astra/issues/355.

### Debug Output
[Output.](https://gist.github.com/jonathanjuursema/4723c2271e441b980fac9cd083c80f89#file-terraform_debug_output-txt)

### Panic Output
No panic observed.

### Expected Behavior
For Terraform to import the table into the state successfully.

### Actual Behavior
The `terraform import` command returned the error shared under Debug Output.

### Steps to Reproduce
Identical to https://github.com/datastax/terraform-provider-astra/issues/355.

### Important Factoids
Names have been obfuscated for security. I can provide full details via a private channel.

Via direct string comparison, I've verified that the database ID, keyspace name and table name in the import command exactly match the resources I observe in the Astra Portal:
- The database ID is identical to the one shown in the Astra Portal Database view.
- The keyspace shows up under "Keyspaces" in that database's Database view.
- I can successfully `USE keyspace_name; DESCRIBE TABLE table_name;`.

### References
I did a quick search for similar issues, but found none.

**Comment by pgier** [comment]
Thanks for the additional info, at least we can see the error now.  Seems like it's getting a bad database hostname for some reason.
```
│ Error: error getting table (not retrying) err: Get "https://08a*****-****-****-****-**********013-.apps.astra.datastax.com/api/rest/v2/schemas/keyspaces/keyspace_name/tables/table_name": GET https://08a*****-****-****-****-**********013-.apps.astra.datastax.com/api/rest/v2/schemas/keyspaces/keyspace_name/tables/table_name giving up after 1 attempt(s): Get "https://08a*****-****-****-****-**********013-.apps.astra.datastax.com/api/rest/v2/schemas/keyspaces/keyspace_name/tables/table_name": dial tcp: lookup 08a*****-****-****-****-**********013-.apps.astra.datastax.com: no such host
```

**Comment by pgier** [comment]
I think maybe this feature never worked.  The problem is that when trying to import a table, the terraform plugin doesn't know what region the database is in.  I created another PR to hopefully fix this: #368

**Comment by jonathanjuursema** [comment]
Awesome, thanks for the assessment and (hopefully) fix. I'm more than happy to test this as soon as it's in the release, but I can imagine that we might want to test this before release.

I'm currently not sure how to test this in our environment without it being in a released version, but I'll try to figure that out later today if I have some time. :)
